components/header.tsx

```
@@ -145,40 +152,75 @@ const Header: React.FC = () => {
+                    <span className={styles["user-name"]}>
+                      {user.displayName}
+                    </span>
+                    <svg
+                      className={`ml-1 w-4 h-4 transition-transform ${
+                        dropdownOpen ? styles["rotate-180"] : styles["rotate-0"]
+                      }`}
+                      fill="none"
+                      stroke="currentColor"
+                      strokeWidth="2"
+                      viewBox="0 0 24 24"
+                    >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)